packages/oid4vci-demo-frontend/src/pages/SSIInformationFromVPRequest/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react'
+import { Text } from '../../components/Text'
+import style from '../../components/Text/Text.module.css'
+import DeepLink from '../../components/DeepLink'
+import { useTranslation } from 'react-i18next'
+import { AuthorizationResponsePayload } from '@sphereon/did-auth-siop'
+import { useNavigate } from 'react-router-dom'
+import MemoizedAuthenticationQR from '../../components/AuthenticationQR'
+import SSIPrimaryButton from '../../components/SSIPrimaryButton'
+import { useMediaQuery } from 'react-responsive'
+import { NonMobile } from '../../index'
+import { useFlowRouter } from '../../router/flow-router'
+import { useEcosystem } from '../../ecosystem/ecosystem'
+import { SSIInformationFromVPRequestPageConfig } from '../../ecosystem/ecosystem-config'
+
+export interface QRCodePageProperties {
+  setData: React.Dispatch<React.SetStateAction<AuthorizationResponsePayload | undefined>>
+}
+
+export default function SSIInformationFromVPRequestPage(): React.ReactElement | null {
```

Comment:
Is this now a VCI page QR or is is a VP page. The name suggests VP, but this PR is seeking to have a single page VCI as used in previous demo's next to the new multicard issuer page

Reply:
this is indeed a VP page. we had the old issuer page, and I'm using that in our triall and fmdm demos. this page is based on our previous `with a wallet` page that went under huge changes for the recent demos